manifests/action_deploy_artifacts.pp

```diff
@@ -166,7 +198,17 @@
             $artifact_details_content['children'].each | Integer $dispatcher_conf_template_index, Hash $dispatcher_conf_template| {
               file { regsubst("${dispatcher_conf_dir}/${dispatcher_conf_template[name]}", '.epp', '', 'G'):
                 ensure  => file,
-                content => epp("${path}/${artifact[name]}/dispatcher-conf-templates/${dispatcher_conf_template[name]}", {conf_dir => $dispatcher_conf_dir}),
+                content => epp("${path}/${artifact[name]}/dispatcher-conf-templates/${dispatcher_conf_template[name]}", {
```

Comment:
The params hash can be declared as a variable and should be injected to all templates, so all templates will have access to the same set of parameters.
Even though the platform's default and sample templates only use certain variables, users might have more complex use cases the require more params.

Reply:
done